Source/Core/Core/AchievementManager.cpp

```diff
@@ -987,6 +988,12 @@ void AchievementManager::LoadGameCallback(int result, const char* error_message,
     WARN_LOG_FMT(ACHIEVEMENTS, "Failed to load data for current game.");
     OSD::AddMessage("Achievements are not supported for this title.", OSD::Duration::VERY_LONG,
                     OSD::Color::RED);
+    if (instance.m_dll_found && result == RC_NO_GAME_LOADED)
+    {
+      // Allow developer tools for unidentified games
+      rc_client_set_read_memory_function(instance.m_client, MemoryPeeker);
+      instance.m_system.store(&Core::System::GetInstance(), std::memory_order_release);
```

Comment:
I don't see what this would be needed for... MemoryPoker uses it, but not MemoryPeeker. I guess having it here is fine though.